src/styles/components/statusbar.less

```diff
@@ -0,0 +1,49 @@
+.status-bar {
+  display: inline-flex;
+  background-color: @gray;
+  margin-right: @base-spacing-unit * 2/3;
```

Comment:
Decided to only use the spacing on `large` screens since the text which needs the margin is only shown on large screens.